src/utility/upload.ts

```diff
@@ -61,13 +66,24 @@ function isAcceptableFile(file: File, accept: string) {
 
   const wildcards = ['audio/*', 'image/*', 'video/*']
 
-  return accept.split(',').some(acceptToken => {
-    if (acceptToken.startsWith('.')) {
-      // tokens starting with a dot represent a file extension
-      return file.name.endsWith(acceptToken)
-    } else if (wildcards.includes(acceptToken)) {
-      return file.type.startsWith(acceptToken.substr(0, acceptToken.length - 1))
-    }
-    return file.type === acceptToken
-  })
+  return accept
+    .replace(/\s+/, '')
+    .toLowerCase()
+    .replace('.jpeg', '.jpg')
+    .replace('/jpeg', '/jpg')
+    .split(/,/)
+    .some(acceptToken => {
+      if (acceptToken.startsWith('.')) {
+        // tokens starting with a dot represent a file extension
+        return file.name
+          .toLowerCase()
+          .replace(/\.jpeg$/, '.jpg')
+          .endsWith(acceptToken)
+      } else if (wildcards.includes(acceptToken)) {
+        return file.type
+          .toLowerCase()
+          .startsWith(acceptToken.slice(0, acceptToken.length - 1))
+      }
+      return file.type.toLowerCase().replace('/jpeg', '/jpg') === acceptToken
+    })
```

Comment:
Your replacements aren't global so will only replace the first instance of each.

Regarding JPEG/JPG; JPEG is the only valid MIME type but after playing with Chrome I see browsers are really lenient. If this is the same as other browsers then lenient matching here makes sense and will save devs from dealing with jpg/jpeg typos that don't matter. On the other hand, I would lean towards stricter tests and writing more lenient code (`accept=".jpg, .jpeg"`).

Suggestion with global replacements and lenient matching:
```js
// When matching files, browsers ignore case and consider jpeg/jpg interchangeable.
function normalize(nameOrType: string) {
  return nameOrType.toLowerCase().replace(/\bjpg\b/g, 'jpeg')
}

function isAcceptableFile(file: File, accept: string) {
  if (!accept) {
    return true
  }

  const wildcards = ['audio/*', 'image/*', 'video/*']

  return normalize(accept)
    .trim()
    .split(/\s*,\s*/)
    .some((acceptToken) => {
      // tokens starting with a dot represent a file extension
      if (acceptToken.startsWith('.')) {
        return normalize(file.name).endsWith(acceptToken)
      } else if (wildcards.includes(acceptToken)) {
        return normalize(file.type).startsWith(acceptToken.replace('*', ''))
      }
      return normalize(file.type) === acceptToken
    })
}
```

Reply:
Great suggestions! I incorporated changes but made a smaller adjustment to the regex---just to make it a little stricter just in case `jpg` also appears in the middle of a file or folder name.